HULKs/hulk
# Rework feet detection

## Introduced Changes

Rewrites most of the feet detection, only focusing detection on the bottom camera.

## ToDo / Known Issues

none

## Ideas for Next Iterations (Not This PR)

none

## How to Test

- Deploy a robot or load the `robot_detection` world in webots
- Observe overlays in both map and image panel for the feet detection
